uptrain/framework/checks.py

```diff
@@ -187,7 +208,7 @@ def dict(self) -> dict:
         return {
             "source": to_py_types(self.source),
             "preprocessors": [to_py_types(op) for op in self.preprocessors],
-            "checks": [to_py_types(check) for check in self.checks],
+            "checks": [to_py_types(check) for check in self.checks + [self._consolidated_check]],
```

Comment:
This isn't needed. If you serialise and deserialize a checkset repeatedly, this will just keep appending a consolidated check to the original list of checks for the Checkset. 

Rather, the streamlit file can just add the consolidated check to the list of checks it plots here. (https://github.com/uptrain-ai/uptrain/blob/97134579000ab7b2bff1ffb28f6d2511061f57c2/uptrain/dashboard/st_helpers.py#L47)

Reply:
makes sense. Updated the PR